user.go

```diff
@@ -0,0 +1,23 @@
+package mockoidc
+
+// User defines the expected user values
+type User struct {
+	Email             string   `json: email`
+	Phone             string   `json: phone`
+	PreferredUsername string   `json: preferred_username`
+	Address           string   `json: address`
+	Groups            []string `json: groups`
+	EmailVerified     bool     `json: email_verified`
+}
+
+var (
+	// DEFAULT_USER is a User struct that can be used wherever a user is needed
+	DEFAULT_USER = User{
```

Comment:
I think this is best as a function: `func DefaultUser() *User`

That stuff can call like this `mockoidc.DefaultUser()`

We'd be recreating the same options each time in favor of semi-immutability. This being a variable I think runs the risk of pollution thinking about it more.